### ISSUE TITLE
WinAppDriver FindElementsByXPath sometimes fails when the singular form succeeds (and vice versa).

### ISSUE THREAD

**Issue by kfertitta** [microsoft/WinAppDriver#1846]
This is something I see a lot, but without any common pattern I'm able to infer.  The exact same XPath, when used in the singular `FindElementByXPath` version works, whereas it returns no results when used in the plural version `FindElementsByXPath`.  In other cases, the opposite is true -- the plural version returns results and the singular version throws a "not found" exception.

**Comment by kfertitta** [comment]
@Shakevg Thank you for the reply.  Unfortunately, that does not seem to be the issue, as I have, indeed, tried various waiting scenarios, including the `DefaultWait` mechanism in WinAppDriver.

For example, the following does not return any results with `FindElementsByPath`, where as the singular version does:

```
var wait = DefaultWait(....) { Timeout = TimeSpan.FromSeconds(10) };

var element = wait.Until(s => s.FindElementsByXPath(Locator).FirstOrDefault());
```

Is nobody else seeing such behavior?  Happens a lot to us.

**Comment by Shakevg** [comment]
[kfertitta](https://github.com/kfertitta) I didn't face such issues, maybe it is a bug.
You can use FindElementByXPath with DefaultWait for a specific case or try to change XPath as a workaround or try to search parent first, then child items with more simple XPath.

**Comment by Nativu5** [comment]
How about this #1284?

**Comment by kfertitta** [comment]
@Nativu5 , Yes indeed.  A quick check reveals that turning off the "ms:experimental-webdriver" option makes `FindElementsByXPath` work again.  Thanks for that.

I guess the question is:  what breaks/changes by turning this options off?
